src/Subscriber/CheckPasswordSubscriber.php

```diff
@@ -24,6 +26,7 @@ public static function getSubscribedEvents(): array
         return [
             GenericPageLoadedEvent::class => 'onPageLoaded',
             HttpCacheHitEvent::class => 'onCachedPageLoaded',
+            CoreHttpCacheHitEvent::class => 'onCachedPageLoaded',
```

Comment:
add deprecated-comment for the 6.5er code so we know what to delete in 3.0.0 :-)